cluster/addons/rbac/cluster-loadbalancing/glbc/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+# This was adapted from the canonical ClusterRole for GLBC can be found at:
+# https://github.com/kubernetes/ingress-gce/blob/master/docs/deploy/resources/rbac.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:glbc
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups: [""]
+  resources: ["secrets", "endpoints", "services", "pods", "nodes", "namespaces", "configmaps", "events"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
+- apiGroups: ["extensions", "networking.k8s.io"]
+  resources: ["ingresses"]
+  verbs: ["get", "list", "watch", "update"]
+- apiGroups: ["extensions", "networking.k8s.io"]
+  resources: ["ingresses/status"]
+  verbs: ["update"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
```

Comment:
giving a controller CRD create permissions is not a great pattern. if these write permissions are really required, can we add a comment referencing why (ideally pointing at the code that requires it so we can track when it's no longer needed), and a TODO to split the CRD manifest creation out of the glbc controller itself

Reply:
Code to create CRD's is here: https://github.com/kubernetes/ingress-gce/blob/master/cmd/glbc/main.go#L93

Reply:
Added an issue and a TODO: https://github.com/kubernetes/ingress-gce/issues/744